js/index.js

```diff
@@ -1,7 +1,64 @@
 // Iteration 1: Names and Input
 
+let hacker1 = "Vera"
+
+console.log("The driver's name is " + hacker1)
+
+let hacker2 = "Santi"
+
+console.log("The navigator's name is " + hacker2)
 
 // Iteration 2: Conditionals
 
+if(hacker1.length > hacker2.length) {
+    console.log("The driver has the longest name, it has " + hacker1.length + "characters.")
+  } else if (hacker2.length > hacker1.length){
+    console.log("It seems that the navigator has the longest name, it has " +  hacker2.length + " characters.")
+  } else {
+    console.log("Wow, you both have equally long names, " + hacker1.length + " characters!.")
+  }
+
+// Iteration 3: Loops
+
+function allCapsSpaced(str) {
+    let initString = ""
+  
+    for(i=0; i < hacker1.length; i++){
+      initString = initString + str[i].toUpperCase() + " "
+    }
+    initString.trim()
+    return initString
+  }
```

Comment:
La lógica ea buena 👍 
cuidado porque la función tiene un parámetro "str" pero dentro de la función usas la variable "hacker1" en la condición del bucle for.
Bien por usar el método trim(), pero recuerda que este método no modifica directamente la cadena. Podrías guardarlo en una variable y devolver esta variable, o directamente `return initString.trim()`